app-frontend/src/app/pages/projects/edit/annotate/annotate.controller.js

```diff
@@ -239,46 +242,31 @@ export default class AnnotateController {
             this.annoToExport.features.push({
                 'properties': {
                     'id': id,
-                    'label': 'Label name...',
-                    'description': 'Description...'
+                    'label': '',
+                    'description': ''
                 }
             });
         } else {
             this.annoToExport.features.push({
                 'properties': {
                     'id': id,
                     'label': this.filterLabel.name,
-                    'description': 'Description...'
+                    'description': ''
                 }
             });
         }
         this.disableToolbarAction = true;
-        this.scrollId = id;
+        this.scrollToItem(id);
     }
 
     onShapeCreated(shapeLayer) {
         this.getMap().then((mapWrapper) => {
             let geojsonData = shapeLayer.toGeoJSON();
-            this.createEditableAndDraggableDrawLayer(mapWrapper, geojsonData.geometry, true);
+            this.createEditableAndDraggableDrawLayer(mapWrapper, geojsonData.geometry);
             this.addNewAnnotationIdToData();
         });
     }
 
-    /* eslint-disable no-underscore-dangle */
-    onCancelAddAnnotation(id) {
-        this.disableSidebarAction = false;
-        this.disableToolbarAction = false;
-        this.isCreating = false;
-
-        if (this.editHandler && this.editHandler._enabled) {
-            this.editHandler.disable();
-        }
-
-        _.remove(this.annoToExport.features, (f) => f.properties.id === id);
-        this.getMap().then((mapWrapper) => mapWrapper.deleteLayers('draw'));
-    }
-    /* eslint-enable no-underscore-dangle */
-
     createGeojsonFromDrawLayerWithMetadata(mapWrapper, id, label, description) {
```

Comment:
seems a little verbose - do we ever create geojson from a draw layer _without_  metadata?